tools/codegen/genwebsitejson.py

```diff
@@ -57,6 +57,11 @@ def platform_for_spec(path):
     platforms that table will work on. In the event that no match is found, it
     will be assumed that the table is found on all platforms.
     """
+    # Special case for yara_events which doesn't follow the typical directory
+    # structure indicating supported platforms.
+    if path.endswith("yara_events.table"):
+        return ["darwin", "linux"]
```

Comment:
I think we should just remove the special cases and actually move the `.table` in the existing `linux`, `posix` etc (depending on where it has to go). This would go under `posix`.
`yara`, `utility` and `sleuthkit` seem to be the only remaining special cases

Reply:
I agree actually. I don't see much benefit of the special case directories. If they aren't important to the build process in some way then that would be my preference.